sphinx_gallery/gen_rst.py

```diff
@@ -1281,7 +1294,7 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf, seen_backrefs=No
 """
 
 
-def rst_blocks(script_blocks, output_blocks, file_conf, gallery_conf):
+def rst_blocks(script_blocks, output_blocks, file_conf, language, gallery_conf):
```

Comment:
I doubt anyone has been using these functions (and they probably shouldn't be) but just in case to minimize pain how about:
```suggestion
def rst_blocks(script_blocks, output_blocks, file_conf, gallery_conf, *, language="python"):
```